pkg/agent/openflow/pipeline.go

```diff
@@ -1857,16 +1861,15 @@ func (f *featureNetworkPolicy) conjunctionActionDenyFlow(conjunctionID uint32, t
 	}
 
 	if enableLogging || f.enableDenyTracking || disposition == DispositionRej {
```

Comment:
I am thinking shall we use the same packetIn strategy (meter->packetIn) for `DispositionRej` as `enableLogging`. For example, if meter is supported, it is possible to drop the packet which is requiring reject action before it is sent to antrea-agent, then the client would fail to receive the response packet. What is your thought? @Dyanngg @GraysonWu @tnqn

Reply:
Hope I understand your question. In my viewpoint, we could use the same strategy for rejection. But the reason for using this strategy on rejection is a little bit different than on logging.
If I understand correctly, using it on logging is mainly focused on addressing the issue that packet-in rate-limiting is blocking allowed traffic. We want the allowed traffic could bypass the meter and apply meter->packetin on another path with a cloned packet.
As for rejection, the reject responses are generated on antrea-agent. So no matter doing packetin on another path or not, as long as we have meters on packet-in messages, which is imperative, it is possible that reject responses may not be transmitted back to the client in the case of over-limit requests. 
The next step of the original path of packets that should be rejected is to be dropped in `EgressMetricTable`. So separating the meter->packetin to another path could make the dropped packets metric in `EgressMetricTable` more accurate by including the number of packets that are dropped by the meter before.

BTW, it would be beneficial to include a concise introduction in the PR description to outline the problem being addressed from this PR. Thank you.

Reply:
I may misunderstand your question. If you were asking if we could remove the meter on the reject packet-in. In my viewpoint, it is imperative to implement meters for all packet-in messages. While it is acceptable to acknowledge that certain reject responses may not be transmitted back to the client in the case of over-limit requests, we must prioritize preventing the overwhelming influx of packet-in messages to the antrea-agent.

Reply:
Yes, my ask is about if we should remove teh meter on reject packet-in or not. Let's wait for others' comment.

Reply:
I feel the reject response is just best-effort. When a request is not allowed, it's kind of undesired, and I think it's not important to achieve 100% reject responses to such accesses by sacrificing the limited resources which should be used to serve the desired traffic. And given the fact the reject will only be missing when there is likely a undergoing DOS, we could drop the over-limit requests.

Reply:
Thanks @GraysonWu @tnqn for sharing the thoughts. Since you two have the similar opinions, I would leave the current implementation not changed.